src/models/custom-wildcard-field.ts

```diff
@@ -0,0 +1,9 @@
+import {ExpandedType} from 'compassql/build/src/query/expandedtype';
+
+export interface CustomWildcardField {
+  fields: string[];
```

Comment:
This won't be compatible with how ShelfFieldDef is structured. 

This should be `field: Wildcard<string>` so you can drag and drop wildcard field and it works right away.

Reply:
Let's punt on this to see if it's a good idea in the next PR.